add value example and Reference to metric (#240)
megaease/easeagent: doc/prometheus-metric-schedule.md

## Common label

| label name    |    Value Example    | Description                                                                                                |
|:--------------|:-------------------:|:-----------------------------------------------------------------------------------------------------------|

> | MetricSubType |      "DEFAULT"      | The enum MetricSubType value: `DEFAULT,ERROR,CHANNEL,CONSUMER,PRODUCER,CONSUMER_ERROR,PRODUCER_ERROR,NONE` |

Reviewer: All the Value Example can use `value` to replace "Value". use Reverse single quote to replace the quote.

Author: as you wish

Reviewer: `DEFAULT,ERROR,CHANNEL,CONSUMER,PRODUCER,CONSUMER_ERROR,PRODUCER_ERROR,NONE`
Add more space
`DEFAULT, ERROR, CHANNEL, CONSUMER, PRODUCER, CONSUMER_ERROR, PRODUCER_ERROR, NONE`

Author: > `DEFAULT,ERROR,CHANNEL,CONSUMER,PRODUCER,CONSUMER_ERROR,PRODUCER_ERROR,NONE` Add more space `DEFAULT, ERROR, CHANNEL, CONSUMER, PRODUCER, CONSUMER_ERROR, PRODUCER_ERROR, NONE`

You are such a careful man. done!

| application_jdbc_statement_p999   | double  | TP99.9: The JDBC method execution duration in milliseconds for 99.9% user.                            |

#### Dedicated label
| Label Name | Essential |           Value Example            | Description                                                                                                                                                                                                                                         |
|:-----------|:---------:|:----------------------------------:|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | signature  |   true    | "440580e6c3215eceb4ef992d60adde9a" | Executed JDBC method signature. By default, It is md5 of sql like "select * from data". It can be restored to sql by turning off the switch: `plugin.observability.jdbc.sql.compress.enabled=false`. [Details](./user-manual.md#tracing-and-metric) |

Reviewer 2: ```suggestion
| signature  |   true    | "440580e6c3215eceb4ef992d60adde9a" | Executed JDBC method signature. By default, It is an MD5 of SQL like "select * from data". It can be SQL statement via turning off the switch: `plugin.observability.jdbc.sql.compress.enabled=false`. [Details](./user-manual.md#tracing-and-metric) |
```
